docs/index.rst

```diff
@@ -47,60 +47,43 @@ After running the cookiecutter command, simply follow the command line instructi
 to compete setting up your repository. Add some Jupyter Notebooks, configure your
 environment and push the whole thing to GitHub.
 
-Configuring dask-kubernetes
-~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Configuring Dask
+~~~~~~~~~~~~~~~~
 
-`Dask-kubernetes`_ is a library designed to deploy dask-distributed using
-Kubernetes_. When using dask-kubernetes on binder, you need to specify some
-basic configuration options. An example of a file named `.dask/config.yaml`.
+The Pangeo Binder is configured to include a Dask Gateway server, which allows
+users to create Dask Clusters for distributed computation. To create the clusters,
+**you must include the dask-gateway in your environment file**.
 
-.. .dask/config.yaml
+.. code-block:: yaml
 
-::
+   # binder/environment.yml
+   channels:
+     - conda-forge
+   dependencies:
+     - dask-gateway
+     - dask-labextension
```

Comment:
perhaps we just point to the `pangeo-notebook` metapackage?

Reply:
Seems reasonable, I'll try that out.